frontend/src/components/home/About.js

```diff
@@ -1,7 +1,25 @@
+import { Button, Typography } from '@mui/material'
 import React from 'react'
+import { Box } from '@mui/material'
 
 function About() {
-    return <div>-about component placeholder for home page-</div>
+    return <div>
+        <Box sx={{ textAlign: "right", padding: "100px" }}>
+            <Typography variant='h2' sx={{ fontFamily: "Lato", color: "primary.main", fontWeight: "medium" }}>
+                About Us
+            </Typography>
+            <br></br>
+            <Typography variant='h5' sx={{ fontFamily: "Lato" }}>
+                Blueprint is a student-run organization that creates technological <br></br>solutions for nonprofit organizations.<br></br> <br></br>
+
+                Blueprint is a multinational organization founded in 2013 at the <br></br>University of California, Berkeley. Our UofA Chapter is the fourth <br></br>established chapter in Canada and the first one in Western Canada.
```

Comment:
I recommend using Box to limit the width of the text instead of manually using <br/> to break text. The text break will not be exactly like the mockup but it will be more adaptive to the screen size

MUI's margin top (mt) can also be used to create a line break.